tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -155,7 +155,105 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Favorite Resource Cards
+    [Tags]    ODS-389
+    Click Link    Resources
+    Sleep    5s
+    Click Element    //div[@class="pf-c-toolbar__content-section"]/div[2]/div/button
```

Comment:
Lines 162-164 could be a new keyword `Sort Resources By Name` or similar in ODHDashboard.robot

Reply:
Maybe you can use the keyword `Change The Sort` introduced in this PR

Reply:
Done